TriforkSwiftExtensions/TriforkSwiftExtensionsTests/UIViewExtensionTests.swift

```diff
@@ -11,16 +11,14 @@ import XCTest
 
 class UIViewExtensionTests: XCTestCase {
     
-    func testCircle() {
```

Comment:
hmm.. I still think we should have tests for functionality even it is deprecated. 

It should be removed when the function is removed?

Reply:
Well yeah, but it causes errors because thee function is deprecated 😜 
I think it is okay to remove the error, because we officially do not support it anymore, when it is marked as deprecated.